netci.groovy

```diff
@@ -5,6 +5,35 @@ import jobs.generation.Utilities;
 def project = GithubProject
 def branch = GithubBranchName
 
+class WcfUtilities
+{
+    def wcfRepoSyncServiceCount = 0 
+    def addWcfOuterloopTestServiceSync(def job, String os) { 
+        wcfRepoSyncServiceCount++
+    
+        job.with { 
+            parameters {
+                stringParam('WcfRepoSyncServiceUrl', "http://wcfcoresrv2.cloudapp.net/PRService${wcfRepoSyncServiceCount}/pr.ashx", 'Wcf OuterLoop Test PR Service Uri')
+            }
+        }
+        if (os.toLowerCase().contains("windows")) {
+            job.with { 
+                steps {
+                    batchFile(".\\src\\System.Private.ServiceModel\\tools\\setupfiles\\sync-pr.cmd %WcfRepoSyncServiceUrl%")
+                }           
+            }
+        } else {
+            job.with { 
+                steps {
+                   shell("HOME=\$WORKSPACE/tempHome ./src/System.Private.ServiceModel/tools/setupfiles/sync-pr.cmd \$WcfRepoSyncServiceUrl")
```

Comment:
Is this supposed to be pr.**sh**? Ideally, we can test this for Linux too before merging.